### ISSUE TITLE
Feedback on new website design

### ISSUE THREAD

**Issue by chris48s** [DemocracyClub/Website#52]
New website looks great, but inevitably I've spotted a few minor points. Some of these may be a bit subjective, but I think they are mostly valid.

## Data page is not linked in menu.

https://democracyclub.org.uk/data/ exists and is linked from projects page, but there is no menu item for it.

## Signup form looks a bit odd

Signup form looks like this:
![screenshot at 2017-02-23 20 51 03](https://cloud.githubusercontent.com/assets/6025893/23279734/4b766e5c-fa0e-11e6-9be0-70d856657d0c.png)

I think it would be nicer if it was either like this:
![suggestion1](https://cloud.githubusercontent.com/assets/6025893/23279743/51430714-fa0e-11e6-94fd-d3eb6cd507ab.png)

..or like this:
![suggestion2](https://cloud.githubusercontent.com/assets/6025893/23279752/56097a80-fa0e-11e6-8bf0-80bddca0214b.png)

## Missing whitespage on blog page

![screenshot at 2017-02-23 20 58 53](https://cloud.githubusercontent.com/assets/6025893/23279761/6fe4b136-fa0e-11e6-8c7a-591fa4ca8975.png)

## Menu is in odd place on 'about wheredoivote'

Menu is here:

![screenshot at 2017-02-23 21 05 22](https://cloud.githubusercontent.com/assets/6025893/23279770/7be4bcf6-fa0e-11e6-8a00-017cdd58dc86.png)

Would it look more natural lined up with the top of the screenshot, like this?

![suggestion](https://cloud.githubusercontent.com/assets/6025893/23279774/7fe04e6a-fa0e-11e6-8f88-82cef5df717c.png)

## Copy on /projects is inconsistent

This probably isn't new, but I've just spotted it:

On https://democracyclub.org.uk/projects/ we're saying 'Where Do I Vote' and then 'WhoCanIVoteFor'. We should pick 'Seperate Words' or 'WholeSentenceMashedTogether' and brand them consistently.

Also, the title link for WhereDoIVote is the 'about page', but for WhoCanIVoteFor it is whocanivotefor.co.uk - should the title link be wheredoivote.co.uk for consistency? Copy might need some re-jigging to accommodate.

## Text is a bit cramped in table on polling stations tech specs

![screenshot at 2017-02-23 21 20 14](https://cloud.githubusercontent.com/assets/6025893/23279833/c0bd1c06-fa0e-11e6-8af9-882340ab8ed5.png)

Needs some padding round the borders to let text 'breathe' a bit, or maybe feint lines or different cell alignment for the left column. It isn't clear enough on first glance which description applies to each standard.

**Comment by chris48s** [comment]
Actually, email signup form should move to multi-line layout as single line layout goes weird at mobile res:
![screenshot at 2017-02-23 21 36 44](https://cloud.githubusercontent.com/assets/6025893/23280282/791281dc-fa10-11e6-8425-d484f7570486.png)

Also, side menu on polling stations about page maybe needs more work as the div disappears to a strange place at low res:
![screenshot at 2017-02-23 21 37 26](https://cloud.githubusercontent.com/assets/6025893/23280316/9ddcc838-fa10-11e6-81c8-db28848c5132.png)

Maybe this should move to _above_ the main content when the screen isn't wide enough rather than below?

**Comment by JoeMitchell** [comment]
### 8. Non-mailing list stuff happening in the footer. 
![democracy_club_footer](https://cloud.githubusercontent.com/assets/8096305/23519782/118ec740-ff70-11e6-921c-8e73b482daf2.jpg)

**Comment by JoeMitchell** [comment]
### Returning to Chris' 2. on the sign up form...

![image](https://cloud.githubusercontent.com/assets/8096305/23519949/beddf4de-ff70-11e6-9b25-74ec1faeb242.png)

1. The text in the input field should be the same size and font as "Name", "Email". 

2. The blue button should have white text. 

3. And be the same vertical size as the input boxes. 

4. And I'm going to make a separate thing about 'buttons'. See # 9!

**Comment by JoeMitchell** [comment]
### 9. Buttons!

I'm fairly sure each of these should have had it's own issue. This is going to be a nightmare! Anyway. Buttons. We all love 'em. But our buttons aren't very buttony. 

I quite like the drop shadow which deepens here: 
![GWR](http://g.recordit.co/cPSOjmi1Bm.gif)

But GOV.UK probably wins — every stage is good! The black line, the darkening for mouse-over, then the yellow border and black-line-disappears upon click. Lovely stuff.
![GOVUK](http://g.recordit.co/EIsvZNCm4O.gif)

**Comment by symroe** [comment]
Right-o, loads of stuff done. Bit hard to track if I've done it all, can someone review?

I've left "Copy on /projects is inconsistent", as that's something I think @JoeMitchell is looking at.

**Comment by chris48s** [comment]
All looks loads better, except for 2 issues:

First one:
![screenshot at 2017-03-08 18 24 38](https://cloud.githubusercontent.com/assets/6025893/23717766/102565c8-042d-11e7-88d8-9534e172f66c.png)

Second one:
When I first load it on a device where I've previously visited the site, it looks a bit odd and I have to do a Ctrl+F5 to get the new styles to load properly. I think there are some assets that aren't properly fingerprinted? Tested on 2 devices (laptop + phone).

**Comment by symroe** [comment]
First one: I did that deliberately. Does it look too ugly?
Second: Ok, I'll look in to it.

**Comment by chris48s** [comment]
It looked immediately strange to me, but maybe I am too close to it and had preconceived notions of what it should look like.. maybe get some feedback from others who can be more objective
